cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -18,7 +18,7 @@
     # Do not assume that the interpreter is always named like
     # ld-linux-x86_64.so, as on some architectures there can be a version after
     # the .so suffix, eg. ld-linux-aarch64.so.1
-    /{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}ld-*.so* mrix,
+    /{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}ld{-*,64}.so* mrix,
```

Comment:
The rule should be correct for riscv too, per @xnox's suggestion I looked at the build logs and we have:
```
lrwxrwxrwx root/root         0 2021-09-28 01:38 ./lib/ld-linux-riscv64-lp64d.so.1 -> riscv64-linux-gnu/ld-linux-riscv64-lp64d.so.1
drwxr-xr-x root/root         0 2021-09-28 01:38 ./lib/riscv64-linux-gnu/
-rwxr-xr-x root/root    121520 2021-09-28 01:38 ./lib/riscv64-linux-gnu/ld-linux-riscv64-lp64d.so.1
```
so it even matched the old rule.